app/helpers/posts_helper.rb

```diff
@@ -0,0 +1,2 @@
+module PostsHelper
+end
```

Comment:
If you're not putting anything in here, you should be able to remove (and references to it)